<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="5H5BCK3" failures="1" tests="1" name="com.fatura.testcases.SearchPageTest" time="8.883" errors="0" timestamp="2023-01-27T18:53:48 EET" skipped="0">
  <testcase classname="com.fatura.testcases.SearchPageTest" name="Search" time="8.883">
    <failure message="it was failure because the expected result is not https://deliveroo.co.uk/restaurants/liverpool/liverpool-city-centre?fulfillment_method=DELIVERY&amp;geohash=gcmzgftw761c expected [https://deliveroo.co.uk/restaurants/liverpool/liverpool-city-centre?fulfillment_method=DELIVERY&amp;geohash=gcmzgftw761c] but found [https://deliveroo.co.uk/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: it was failure because the expected result is not https://deliveroo.co.uk/restaurants/liverpool/liverpool-city-centre?fulfillment_method=DELIVERY&geohash=gcmzgftw761c expected [https://deliveroo.co.uk/restaurants/liverpool/liverpool-city-centre?fulfillment_method=DELIVERY&geohash=gcmzgftw761c] but found [https://deliveroo.co.uk/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.fatura.testcases.SearchPageTest.Search(SearchPageTest.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Search -->
  <system-out/>
</testsuite> <!-- com.fatura.testcases.SearchPageTest -->
